scripts/set_xtensa_params.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2023 Intel Corporation. All rights reserved.
+
+set -e
```

Comment:
Don't do this in this file as this will affect the caller (like everything else in this script)